darwin/importer/importer.py

```diff
@@ -39,6 +39,8 @@
 from darwin.utils import flatten_list, secure_continue_request
```

Comment:
**NB: Most changes are Black reformatting the file, I've marked the section involved**